script/format

```diff
@@ -9,11 +9,13 @@ _VENV_DIR = _PROGRAM_DIR / ".venv"
 _MODULE_DIR = _PROGRAM_DIR / "speech_to_phrase"
 _TESTS_DIR = _PROGRAM_DIR / "tests"
 _TEST_FILES = [str(p) for p in _TESTS_DIR.glob("*.py")]
+_SCRIPTS_DIR = _PROGRAM_DIR / "scripts"
+_SCRIPT_FILES = [str(p) for p in _SCRIPTS_DIR.glob("*.py")]
 
 context = venv.EnvBuilder().ensure_directories(_VENV_DIR)
 subprocess.check_call(
-    [context.env_exe, "-m", "black", str(_MODULE_DIR)] + _TEST_FILES
+    [context.env_exe, "-m", "black", str(_MODULE_DIR)] + _TEST_FILES + _SCRIPT_FILES
 )
 subprocess.check_call(
-    [context.env_exe, "-m", "isort", str(_MODULE_DIR)] + _TEST_FILES
+    [context.env_exe, "-m", "isort", str(_MODULE_DIR)] + _TEST_FILES + _SCRIPT_FILES
```

Comment:
Why `_SCRIPT_FILES` and not just point it at `"scripts"`

Reply:
This made me realize I had a typo :smile: